shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -141,13 +141,8 @@ static BOOL IsDeepLinkingEnabled(NSDictionary* infoDictionary) {
   }
 }
 
-- (BOOL)application:(UIApplication*)application
-            openURL:(NSURL*)url
-            options:(NSDictionary<UIApplicationOpenURLOptionsKey, id>*)options
-    infoPlistGetter:(NSDictionary* (^)())infoPlistGetter {
-  if ([_lifeCycleDelegate application:application openURL:url options:options]) {
```

Comment:
This method is now shared between two app delegate methods.  Move the method-specific `_lifeCycleDelegate` check up one level to `-application:openURL:options:`.